### ISSUE TITLE
prepareDatabase crash in 5.0.531

### ISSUE THREAD

**Issue by Twas** [regulaforensics/DocumentReader-iOS#26]
Hello,

The code that has been working before started crashing after update to 5.0.531.

Pods and their current versions:
```
DocumentReader - 5.0.531
DocumentReaderOCR - 5.0.1109
```

Crashing code: 
```
documentReader.prepareDatabase(databaseID: "Full") { _, error in
            completion(error)
}
```

Error:

![PassportScan_xcworkspace](https://user-images.githubusercontent.com/2943081/68209153-6f325100-ffdb-11e9-8ba2-df29924b5b79.png)

Console output: 

> DocumentReader was compiled with optimization - stepping may behave oddly; variables may not be available.

Optimization is turned off for debug configuration. App is launched on simulator.

**Comment by vyakimchik** [comment]
Hello. The app crashes due to missed progressHandler. It'll be fixed in the next version of the SDK.

**Comment by vyakimchik** [comment]
Hello. We published the new version (5.1) where the issue was fixed.
